### ISSUE TITLE
Build wheels in the release script

### ISSUE THREAD

**Issue by asmeurer** [Quansight-Labs/ndindex#157]
Make sure wheels are built in the release script and that they don't cythonize.

**Comment by rgommers** [comment]
Is this still relevant? I think publishing directly through CI looks good?

**Comment by asmeurer** [comment]
Yeah I think CI publishing is working here. This is referring to the old "cythonization" that just cythonized the pure Python with no actual cython code.
